### PR TITLE
Handle 404 error

### LINKED ISSUE #490: Failed to fetch validator.
in `renderer.log` there is an error message` Failed to fetch validator` that show us this error is unhandled or error logging is wrongly handled

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #490

### CODE REVIEW

File: src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts [hunk 1]
```diff
@@ -52,7 +53,8 @@ export class CgEth2BeaconStateApi implements ICGBeaconStateApi {
                 case: "snake",
             }) as ICGValidatorResponse;
         } catch (e) {
-            logger.error("Failed to fetch validator", {validatorId: id, error: e.message});
+            if (!e.message.includes('"code":404'))
```
Comment (mpetrunic) [inline src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts:56]: isn't that axios error? It should contain status code property

Comment (mpetrunic) [reply src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts:56]: https://github.com/axios/axios#handling-errors

Comment (BeroBurny) [reply src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts:56]: https://github.com/NodeFactoryIo/ChainGuardian/blob/23a6deb61e834bfc91b951a63260a194c5d6b014/src/renderer/services/api/http/httpClient.ts#L81

inside `HttpClient` is transformed from `AxiosError` to `Error`

Comment (mpetrunic) [reply src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts:56]: Can we modify it to be http error?

Comment (BeroBurny) [reply src/renderer/services/eth2/client/eth2ApiClient/cgEth2BeaconStateApi.ts:56]: we can, but not sure what time its gonna take to refactor everything

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**